docs/content/docs/try-flink-agents/installation.md

```diff
@@ -23,3 +23,109 @@ under the License.
 -->
 
 # Installation
+
+## Install from PyPI
```

Comment:
I think we need a paragraph about the relationship between python and java. Does every one need to build / install both python and java parts? In which order should we the build python and java parts?